packages/react-components/babel-preset-global-context/src/index.ts

```diff
@@ -1,6 +1,7 @@
 import { transformPlugin } from './transformPlugin';
+import type { PluginItem } from '@babel/core';
 
-export default function preset() {
+export default function preset(): PluginItem {
```

Comment:
Noted in PR description:

> @babel/* packages have been bumped to avoid duplicates
> packages/react-components/babel-preset-global-context/src/index.ts have been updated due TS error "types were not portable"

Reply:
yeah that error comes probably because you need to bump all babel family of packages